benchmarks/benchmarks.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (C) 2021 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Mitiq accuracy and timing benchmarks."""
+
+import functools
+
+import networkx as nx
+import numpy as np
+
+import cirq
+from mitiq import benchmarks, pec, zne, Observable, PauliString
+from mitiq.interface import mitiq_cirq
+
+
+compute_density_matrix_noiseless = functools.partial(
+    mitiq_cirq.compute_density_matrix, noise_level=(0.0,)
+)
+benchmark_circuit_types = ("rb", "mirror")
+
+
+def get_benchmark_circuit(
+    circuit_type: str, nqubits: int, depth: int,
+) -> "cirq.Circuit":
```

Comment:
There might be a good reason for the way that this is (and if so, just ignore this comment), but usually, I only see return types quoted like this if it's in reference to returning an object in a class method whose return type is the same as the class. So, for instance:

```python
class Name:
    def get_name_obj -> "Name":
        pass
...
```

In that instance, you can just `from __future__ import annotations` to overcome the need for this quoted return type-hint. 

All this to say I'm not sure if the `cirq.Circuit` bit needs to be surrounded by quotes or not. My initial thought is it doesn't, but again, I could be missing some sound reason as to why this is the case.

Reply:
There's actually no reason for quotes here, I forget why I did that. Will remove.

Reply:
Removed quotes.